docs/catalog/cloud/review-platform-and-service-updates-regularly.md

```diff
@@ -0,0 +1,33 @@
+---
+version: 1.0
+submitted_by: yelghali
+published_date: tbd
+category: cloud
+tags: 
+- compute
+- kubernetes
+- role:cloud-engineer
+- size:medium
+---
+
+# Review platform and service updates regularly
+
+## Description
+Platform updates enable you to use the latest functionality and features to help increase efficiency. Running on outdated software can result in running a suboptimal workload with unnecessary performance issues. New software tends to be more efficient in general.
+
+## Solution
+Upgrade to newer and more efficient services as they become available.
+
+## SCI Impact
+`SCI = (E * I) + M per R`
+[Software Carbon Intensity Spec](https://grnsft.org/sci)
+
+Regarding the SCI equation, regularly updating your platforms and services will impact:
+
+- `E`:  Using the latest functionality and features helps increase energy efficiency.
```

Comment:
Newer doesn't mean more efficient. For example, newer could mean new features that don't get utilized but still are transfered, installed, or impact the Energy use in some unintended way that ends up decreasing the SCI score for the user.